common/xfer/plugin_spec.go

```diff
@@ -7,8 +7,8 @@ import (
 	"sort"
 
 	"github.com/davecgh/go-spew/spew"
-	"github.com/mndrix/ps"
 	"github.com/ugorji/go/codec"
+	"github.com/weaveworks/ps"
```

Comment:
> Generally we don't change the imports when forking something, we just checkout the fork in the original vendored location.

Where have we done this? (honest question). I think it's a bad practice since it's misleading on what libraries you are really using. Also, it may conflict with the gvt tool.